lib/modules/manager/pipenv/artifacts.spec.ts

```diff
@@ -637,27 +637,28 @@ describe('modules/manager/pipenv/artifacts', () => {
     expect(getMatchingHostRule('')).toBeNull();
   });
 
-  it('extracts correct environment variable from credential placeholder', () => {
-    for (const testCase of [
-      { credential: '$USERNAME', result: 'USERNAME' },
-      { credential: '$', result: null },
-      { credential: '', result: null },
-      { credential: '${USERNAME}', result: 'USERNAME' },
-      { credential: '${USERNAME:-default}', result: 'USERNAME' },
-      { credential: '${COMPLEX_NAME_1:-default}', result: 'COMPLEX_NAME_1' },
-    ]) {
+  it.each([
```

Comment:
can you please use the tagged template syntax instead?

- https://github.com/renovatebot/renovate/blob/main/docs/development/best-practices.md#unit-testing